web/app/common/store.test.js

```diff
@@ -0,0 +1,11 @@
+import store from './store';
+
+describe('Store', () => {
+  it('should trigger listener on add comment', () => {
+    const listener = jest.fn();
```

Comment:
Can you add some space between lines, please? Smth like this:

```
const listener = jest.fn();

store.set('comments', []);
store.onUpdate('comments', listener);
store.addComment({id:`new`});

expect(listener).toBeCalledWith([{"comment": {"id": "new"}}]);
```

And so on in the others files.

Reply:
Fixed